### ISSUE TITLE
Incompatibility with PyCharm 4.5.3/4

### ISSUE THREAD

**Issue by la10736** [vsch/idea-multimarkdown#23]
I tried to use the plug-in by PyCharm (4.5.x) and when I try to open my markdown files I have follow exception

    13.47.37 PluginException: javafx/beans/value/ChangeListener : Unsupported major.minor version 52.0 [Plugin: com.vladsch.idea.multimarkdown]

I downloaded and installed it by plugin PyCharm interface and I'm wondering if I missing something (I did just one click and restart PyCharm). I tried it by 4.5.3 and 4.5.4 versions but I suspect that is an 4.5.x related issue.

**Comment by la10736** [comment]
UPDATE:

I tried PyCharm 4.0.6 and the issue was already present. Maybe I missed something....

**Comment by vsch** [comment]
Thank you for reporting this.

This means you are running the IDEA with JDK 1.6 and the plugin was compiled for 1.8. I will double check that because normally I always stick to 1.6 level for compilation.

You can continue to use the previous version which you can download from the [Official JetBrains plugin site Version 1.0.24](https://plugins.jetbrains.com/plugin/download?pr=&updateId=21191).

I'll verify if it was compiled for 1.8 and correct it for the next release so that it will work with JDK 1.6.

**Comment by la10736** [comment]
As far I can understand I'm using JDK 1.7
```
2015-09-09 13:47:16,385 [      0]   INFO -        #com.intellij.idea.Main - ------------------------------------------------------ IDE STARTED ------------------------------------------------------ 
2015-09-09 13:47:16,399 [     14]   INFO -        #com.intellij.idea.Main - IDE: PyCharm (build #PY-141.2569, 28 Aug 2015 00:00) 
2015-09-09 13:47:16,399 [     14]   INFO -        #com.intellij.idea.Main - OS: Linux (3.16.0-31-generic, amd64) 
2015-09-09 13:47:16,399 [     14]   INFO -        #com.intellij.idea.Main - JRE: 1.7.0_79-b14 (Oracle Corporation) 
2015-09-09 13:47:16,399 [     14]   INFO -        #com.intellij.idea.Main - JVM: 24.79-b02 (OpenJDK 64-Bit Server VM) 
2015-09-09 13:47:16,401 [     16]   INFO -        #com.intellij.idea.Main - JVM Args: -javaagent:/usr/share/java/jayatanaag.jar -Xbootclasspath/a:./../lib/boot.jar -Xms128m -Xmx750m -XX:MaxPermSize=350m -XX:ReservedCodeCacheSize=225m -XX:+UseConcMarkSweepGC -XX:SoftRefLRUPolicyMSPerMB=50 -ea -Dsun.io.useCanonCaches=false -Djava.net.preferIPv4Stack=true -Dawt.useSystemAAFontSettings=lcd -Djb.vmOptionsFile=./pycharm64.vmoptions -XX:ErrorFile=/home/damico/java_error_in_PYCHARM_%p.log -Djb.restart.code=88 -Didea.paths.selector=PyCharm40 -Didea.platform.prefix=Python -Didea.no.jre.check=true 
```

By official plugin works as a sharm.

**Comment by vsch** [comment]
New version released. Will run with JDK 1.6.

**Comment by vsch** [comment]
New release available 1.1.2:
- Fix Exception reporting for ScratchFileService not found on older versions. If the check fails then no others will be performed until build of IDE changes.
- Fix WebView Preview Tab not working for 1.8u60 and up. In 1.8u60+ it no longer handles fonts with uri(data:) nor font/woff only .ttf.
- Fix allowing capital X to mark closed tasks. Fix #24
- Fix lack of ComboBox for Build 133.1711, now for earlier builds using List for theme selection instead of ComboBox.
- Change since-build to 133.1711, earliest tested version IDEA 13.0 CE, new Preview is not available but old preview works.

Please let me know if it solves your issue and/or close the issue which will be taken as issue solved.

**Comment by la10736** [comment]
Fixed

2015-09-10 5:53 GMT+02:00 Vladimir Schneider <notifications@github.com>:
